@uportal/esco-content-menu/src/components/Ellipsis.vue

```diff
@@ -4,42 +4,63 @@
 
 <script>
 // Code translation from https://github.com/jypblue/vue-ellipsis
-// intregration due to a problem : https://github.com/jypblue/vue-ellipsis/issues/2
+// intregration due to some problems and customization needs : https://github.com/jypblue/vue-ellipsis/issues/2
 
 export default {
   name: 'Ellipsis',
   props: {
     message: {type: String, default: ''},
-    lineClamp: {type: Number, default: 1},
+    lineClamp: {type: Number, default: 0},
     lineHeight: {type: String, default: '22px'},
     endChar: {type: String, default: '...'},
     endHtml: {type: String, default: ''},
   },
   mounted() {
     this.$nextTick(function() {
-      setTimeout(() => {
-        this.handleSubstrSentence();
-      }, 200);
+      window.addEventListener('resize', this.timedRun);
+      this.timedRun();
     });
   },
   methods: {
+    timedRun() {
+      setTimeout(() => {
```

Comment:
I'm curious why this needs a timeout.
Has the value not settled by the time the event is called?

Reply:
I have a sync problem that I don't know how to solve, the renderer is faster than the browser, I mean that I don't have element properties set when the function is running, i need to wait a bit...

Reply:
Ahhh, you may want https://vuejs.org/v2/api/#Vue-nextTick

Reply:
didn't succeed ! I'm guessing if the problems doesn't come from content-favorite component as all goes well into content-grid... But also there are more levels of parents components onto this part...

Reply:
I'm keeping this timeout but only on resize event where this is needed to let the time to the browser to render some part before some change depending on rendering (size as example) are calculated and applied

Reply:
Can I close this requested change ? Or you need more explanations ?